### PR TITLE
Port print_int to the new backend

### PR DESCRIPTION
**Pull request by kddnewton**
* Implement `LDURSW` on AArch64 to sign-extend a 32-bit memory address into a 64-bit register
* Implement `SXTW` on AArch64 to sign-extend a 32-bit register into a 64-bit register
* Add `Op::LoadSExt` to load a value and sign-extend it into a register
* Port `print_int` to the new backend

### CODE REVIEW

File: yjit/src/asm/arm64/mod.rs [hunk 1]
```diff
@@ -321,6 +321,21 @@ pub fn ldur(cb: &mut CodeBlock, rt: A64Opnd, rn: A64Opnd) {
     cb.write_bytes(&bytes);
 }
 
+/// LDURSW - load a 32-bit memory address into a register and sign-extend it
```
Comment (maximecb) [inline yjit/src/asm/arm64/mod.rs:324]:
```suggestion
/// LDURSW - load a 32-bit integer from memory into a register and sign-extend it
```

**Comment by maximecb** [comment]
Looks good. One thing I would like to see is a test in `yjit/src/backend/tests.rs` that calls `print_int`.

**Comment by noahgibbs** [comment]
Huh. So a test for offset_of went away, but we're still using the crate?

**Comment by noahgibbs** [comment]
Ah, never mind, guess it was duplicated.

**Pull request merged**